app/views.py

```diff
@@ -12,7 +12,7 @@ def print_user_input(query):
     if '=' in query:
         query_container, query_value = query.split('=')
         return 'Your query is {} which is equal to {}'.format(query_container, query_value)
-    return page_not_found(404)
+    return "You've entered an incorrect query. Please check and try again. Input : {}".format(query)
```

Comment:
nice error message! can you be slightly more specific, though? what did the client not do?

what status code is this returning at the moment? the best thing is to add a test that makes sure that a 400 is being returned.